user/docker.md

```diff
@@ -35,10 +32,8 @@ Docker containers built from the same image:
 - a Sinatra application
 - the Sinatra application test suite
 
-After specifying in the `.travis.yml` that the worker is using the Docker
-enabled environment (with `sudo: required` AND `services: - docker`) and is
-using ruby, the `before_install` build step pulls a Docker image from
-[carlad/sinatra](https://registry.hub.docker.com/u/carlad/sinatra/) then runs
+After specifying in the `.travis.yml` that the worker is using the Docker enabled environment (with `services: - docker`) and is
+using ruby, the `before_install` build step pulls a Docker image from [carlad/sinatra](https://registry.hub.docker.com/u/carlad/sinatra/) then runs
```

Comment:
, the `before_install` build step pulls a Docker image from [carlad/sinatra](https://registry.hub.docker.com/u/carlad/sinatra/) then runs